### PR TITLE
Added a link to TableOperations.jl and fixed the link to INTEGRATIONS.md

### PR DESCRIPTION
**Pull request by xinkai-zhou**

### CODE REVIEW

**Review by quinnj — COMMENTED** [review]
thanks!

**Comment by quinnj** [comment]
Hmmm, it seems I'm not able to resolve the conflicts here. @xinkai-zhou could you rebase this on latest master so I can merge?

File: docs/src/index.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ don't need to use Tables.jl directly.
 With that said, don't hesitate to [open a new issue](https://github.com/JuliaData/Tables.jl/issues/new), even
 just for a question, or come chat with us on the [#data](https://julialang.slack.com/messages/data/) slack
 channel with questions, concerns, or clarifications. Also one can find list of packages that supports
-Tables.jl interface in [INTEGRATIONS.md](https://github.com/Arkoniak/Tables.jl/blob/master/INTEGRATIONS.md).
+Tables.jl interface in [INTEGRATIONS.md](https://github.com/JuliaData/Tables.jl/blob/master/INTEGRATIONS.md).
+
+Please refer to [TableOperations.jl](https://github.com/JuliaData/TableOperations.jl) for common table operations such as  ```select, transform, filter``` and  ```map```.
```
Comment (nalimilan) [inline docs/src/index.md:15]: Use a single backtick, and for each function separately. Same in README.

**Comment by quinnj** [comment]
Ok, I cherry-picked @xinkai-zhou 's commits over at https://github.com/JuliaData/Tables.jl/pull/184 and fixed the backticks. Thanks again @xinkai-zhou for the contribution!

**Comment by xinkai-zhou** [comment]
@quinnj Sorry for making it a hassle..

**Comment by quinnj** [comment]
No problem at all; sometimes git/github just get confused w/ things; it was weird that it wouldn't let me resolve the conflict in this PR and I had to open a new one.

**Pull request closed without merging**